tests/unit/includes/gateway/class-omise-payment-internetbanking-test.php

```diff
@@ -8,7 +8,6 @@ public function setUp(): void
     {
         $this->sourceType = 'fpx';
         parent::setUp();
-        // require_once __DIR__ . '/../../../../includes/backends/class-omise-backend-fpx.php';
```

Comment:
Why commented?

Reply:
Don't remember. That's why I removed it.

Reply:
I typically prefer to remove altogether than just comment out.

Reply:
My mistake.